content/client-lib-development-guide/features.textile

```diff
@@ -989,6 +992,12 @@ h3(#activation-state-machine). Activation state machine
 * @(RSH4)@ When an event is fired and a transition from the current state is not defined for such event, the event is put into a queue. Then, whenever a transition happens, an event is dequeued from the queue. If a transition from the new current state is defined for the dequeued event, such transition happens. If not, the event is put back in its place in the queue. E. g. we're @WaitingForDeregistration@, and an event @CalledActivate@ happens. This event will be put in the queue, since there's no transition defined for it. Then, an event @Deregistered@ arrives, causing a transition to @NotActivated@. Now we peek the next item on the queue: @CalledActivate@. Because @NotActivated@ transitions on @CalledActivate@, the event is consumed and the machine transitions.
```

Comment:
@paddybyers This conflicts with https://github.com/ably/docs/pull/399/commits/ca19daac56e1f87132bd22410d12c643906c3212#diff-11900b395df266bc2bbfc1d11bdfc7a0R902 Does it belong to RSH3?

Reply:
What's the conflict?